net-libs/tox/tox-0.1.10.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools systemd user
+
+DESCRIPTION="Encrypted P2P, messaging, and audio/video calling platform"
+HOMEPAGE="https://tox.chat"
+SRC_URI="https://github.com/TokTok/c-toxcore/releases/download/v${PV}/c-toxcore-${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Is there a very good reason why you're renaming the distfile instead of using what upstream uses? I see no real benefit in that, and it is confusing considering that the top directory inside follows the original name.